lexicons/app/bsky/unspecced/getPostThreadV2.json

```diff
@@ -0,0 +1,144 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.unspecced.getPostThreadV2",
+  "defs": {
+    "main": {
+      "type": "query",
+      "description": "(NOTE: this endpoint is under development and WILL change without notice. Don't use it until it is moved out of `unspecced` or your application WILL break) Get posts in a thread. It is based in an anchor post at any depth of the tree, and returns posts above it (recursively resolving the parent, without further branching) and below it (recursive replies, with branching). Does not require auth, but additional metadata and filtering will be applied for authed requests.",
+      "parameters": {
+        "type": "params",
+        "required": ["anchor"],
+        "properties": {
+          "anchor": {
+            "type": "string",
+            "format": "at-uri",
+            "description": "Reference (AT-URI) to post record."
+          },
+          "above": {
+            "type": "integer",
+            "description": "How many levels of parent (and grandparent, etc) to include above the anchor.",
+            "default": 80,
+            "minimum": 0,
+            "maximum": 100
+          },
+          "below": {
+            "type": "integer",
+            "description": "How many levels of replies to include below the anchor.",
+            "default": 6,
+            "minimum": 0,
+            "maximum": 100
+          },
+          "branchingFactor": {
+            "type": "integer",
+            "description": "Maximum of replies to include at each level of the thread, except for the direct replies to the anchor, which are (NOTE: currently, during unspecced phase) all returned (NOTE: later they might be paginated).",
+            "default": 10,
+            "minimum": 0,
+            "maximum": 100
+          },
+          "prioritizeFollowedUsers": {
+            "type": "boolean",
+            "description": "Whether to prioritize posts from followed users. It only has effect when the user is authenticated.",
+            "default": false
+          },
+          "sorting": {
+            "type": "string",
+            "description": "Sorting for the thread replies.",
+            "knownValues": [
+              "app.bsky.unspecced.getPostThreadV2#newest",
+              "app.bsky.unspecced.getPostThreadV2#oldest",
+              "app.bsky.unspecced.getPostThreadV2#top"
```

Comment:
This only needs to be tokens if we want to leave it open to extension by other lexicon namespaces.  It might be a case where we can get by with regular strings.  There is some precedence for this in `threadViewPref.sort`.